[DOCS-12567] Updating inconsistent policy names (#428)
mulesoft/docs-api-manager: modules/ROOT/pages/cors-policy.adoc

= Cross-Origin Resource Sharing
ifndef::env-site,env-github[]
include::_attributes.adoc[]
endif::[]
:keywords: cors, api gateway, gateway, policy

[%autowidth.spread,cols="a,a"]
|===
>s| Policy name | Cross-Origin Resource Sharing
>s| Summary      | Enables access to resources residing in external domains
>s| Category | Compliance
>s| First Mule version available | v4.1.1
.1+>.^s| Returned Status Codes
| No return codes exist for this policy
|===

Cross-Origin Resource Sharing (CORS) is a mechanism by which a web application can access resources that are defined in another domain. Browsers implement this standard by default. The Cross-Origin Resource Sharing policy complies with the https://fetch.spec.whatwg.org/[CORS W3C recommendation] standards.

== Why CORS

> The Cross-Origin Resource Sharing policy enables you to achieve web page security and web integrity in your environment. To know why you might need to apply the Cross-Origin Resource Sharing policy to your backend, you must first understand _origins_, _cookies_, and how they can be manipulated to violate web page integrity.

Reviewer: ```suggestion
The Cross-Origin Resource Sharing policy enables you to achieve web page security and web integrity in your environment. To know why you might need to apply the Cross-Origin Resource Sharing policy to your backend, you must first understand _origins_ and _cookies_, and how they can be manipulated to violate web page integrity.
```

Author: will fix

== How CORS Works

> When your web pages request data, the browser detects whether the request is from within same origin and determines whether to apply the CORS algorithm. If you query data from a web page that is not in your origin, then the Cross-Origin Resource Sharing policy is applied. 

Reviewer: ```suggestion
When your web pages request data, the browser determines whether the request is from within your origin and if not, applies the CORS algorithm.
```

Author: will fix

> The CORS algorithm works on the web server and on the client-side for the web page that requested the information. The client-side algorithm in the Cross-Origin Resource Sharing policy is implemented by:

Reviewer: ```suggestion
The CORS algorithm works on the web server and on the client side for the web page that requested the information. The client-side algorithm in the Cross-Origin Resource Sharing policy is implemented by:
```

Author: will fix

> == FAQs

Reviewer: Per style ref.:

FAQ use in documentation
Going forward, rework legacy FAQs as time allows, and do not create new FAQs.

Anything that you anticipate your audience having questions about belongs in narrative context or within a reference or a troubleshooting topic. Ask your editor for assistance.

For those who are interested in a complete explanation of why FAQs violate the CX goal of crafting purposeful information, the following article is excellent:

https://alistapart.com/article/no-more-faqs-create-purposeful-information-for-a-more-effective-user-experi/

Author: ok, I will open a Gus ticket to weave the FAQs into the rest of the content in the future - thanks Kim
